packages/repository-json-schema/src/build-schema.ts

```diff
@@ -255,11 +262,14 @@ export function getNavigationalPropertyForRelation(
   }
 }
 
-function getTitleSuffix(options: JsonSchemaOptions = {}) {
+function getTitleSuffix<T extends object>(options: JsonSchemaOptions<T> = {}) {
   let suffix = '';
   if (options.partial) {
     suffix += 'Partial';
   }
+  if (options.optional && options.optional.length) {
+    suffix += 'Optional[' + options.optional + ']';
+  }
   if (options.includeRelations) {
     suffix += 'WithRelations';
   }
```

Comment:
Would it be simpler to have the title as:

```
<originalModelName>:<stringified-json-for-options>
```

Reply:
Please note the suffix is used in JSON/OpenAPI Schema titles. These titles are later used by code generators like swagger-codegen to emit interface or class names in TypeScript/Java/C#/etc. 

While it may be easier for us to build the prefix as stringified json, I believe it will provide suboptimal experience for our users and anybody consuming OpenAPI spec generated by LB4 applications.